src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBufferProvider.ts

```diff
@@ -50,6 +51,14 @@ export class TerminalAccessibleBufferProvider extends DisposableStore implements
 		}));
 	}
 
+	onKeyUp(e: IKeyboardEvent): void {
```

Comment:
probably should be `onKeyDown`, but there was some reason I went with this, would need investigation